src/components/Block.jsx

```diff
@@ -20,20 +21,31 @@ module.exports = React.createClass({
     return app.refine('blockTypes').find(i => i.id === block.type)
   },
 
+  setMenuItems(component) {
+    if ('getMenuItems' in component) {
```

Comment:
Not really :P. We could throw an error or let the browser throw an error. Do you have strong feels on giving a super clear error here?